docs/rules/0192/deprecated-method-comment.md

```diff
@@ -0,0 +1,65 @@
+--------------------------------------------------------------------------------
+
+rule: aip: 192 name: [core, '0192', deprecated-method-comment] summary:
```

Comment:
This section isn't formatted the same way as the other rule docs, e.g.

https://raw.githubusercontent.com/googleapis/api-linter/main/docs/rules/0192/only-leading-comments.md

Reply:
Done.